app/src/androidTest/java/com/duckduckgo/app/browser/navigation/WebViewBackForwardListSafeExtractorTest.kt

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2019 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.app.browser.navigation
+
+import android.content.Context
+import android.webkit.WebBackForwardList
+import android.webkit.WebView
+import androidx.test.annotation.UiThreadTest
+import androidx.test.platform.app.InstrumentationRegistry.getInstrumentation
+import org.junit.Assert.assertNull
+import org.junit.Test
+
+class WebViewBackForwardListSafeExtractorTest {
+
+    @UiThreadTest
+    @Test(expected = NullPointerException::class)
+    fun whenCopyBackForwardListCalledTestWebViewThrowsException() {
```

Comment:
Unit tests are often thought of as documentation. The place to check assumptions, how a system should and does behave. This unit test is really confusing in that it suggests that copying the back forward list should throw a NPE. Do we need this? I suggest we delete it.

Reply:
Good point. I wanted the test to protect against someone accidentally changing the safe extractor in a way which no longer internally catches the exception. I think its existence adds value.

Maybe we name it better to be clear it's not expected behaviour that copying the list should throw the NPE?

Reply:
Scratch that last comment; deleting it as per your suggestion.

Reply:
Update: We discussed this over zoom and agreed to delete the additional test